test/mjsunit/simd/float32x4.js

```diff
@@ -942,3 +942,32 @@ function testArrayOfFloat32x4() {
 }
 
 testArrayOfFloat32x4();
+
+function testFloat32x4OveralignedLoad() {
+  var b = new ArrayBuffer(40);
+  var a = new Float32Array(b, 8);
+  var af = new Float64Array(b, 8);
+  for (var i = 0; i < a.length; i++) {
+    a[i] = i;
+  }
+
+  for (var i = 0; i < a.length - 3; i += 2) {
+    var v = SIMD.float32x4.load(af, i / 2);
+    assertEquals(i, v.x);
+    assertEquals(i+1, v.y);
+    assertEquals(i+2, v.z);
+    assertEquals(i+3, v.w);
+  }
+}
+
+testFloat32x4OveralignedLoad();
+
+function testFloat32x4ArrayIndex() {
```

Comment:
This test case is for another bug, correct? Please add it into the corresponding PR.